internal/integration/knot/entities/device.go

```diff
@@ -1,12 +1,27 @@
 package entities
 
+//device state regarding the knot connection
```

Comment:
This comment is not following the pattern of this file.
```suggestion

const (
	// KNoT Protocol constants
	KnotNew           string = "new"
	KnotRegisterReq          = "registerRequest"
	KnotUnregisterReq        = "registerUnrequest"
	KnotRegistered           = "registered"
	KnotDelete               = "delete"
	KnotForceDelete          = "forceDelete"
	KnotoK                   = "readToSendData"
	KnotAuth                 = "authenticated"
	KnotError                = "error"
)
```

Reply:
ok